src/main/webapp/dev/js/main/adminSearch.es6

```diff
@@ -144,4 +144,53 @@ $(document).ready(() => {
     $('#btn-collapse-all-students').on('click', () => {
         adminSearchCollapseAllStudents();
     });
+
+    $('.openEmailApplicationDefaultValues').submit((e) => {
+        e.preventDefault();
+        const entry = $(e.target);
+        const studentEmail = $('input[name=studentEmail]').val();
+        const subjectType = $(entry).find('input[name=subjectType]').val();
+        const courseName = $(entry).find('input[name=courseName]').val();
+        const courseId = $(entry).find('input[name=courseId]').val();
+        const studentName = $(entry).find('input[name=studentName]').val();
+        const sessionStatus = $(entry).find('input[name=sessionStatus]').val();
+
+        const relatedLink = encodeURIComponent($(entry).find('input[name=relatedLink]').val());
+        const googleSignup = encodeURIComponent('https://accounts.google.com/NewAccount');
+        const supportEmail = 'teammates@comp.nus.edu.sg';
```

Comment:
Might be better if it was possible to get this value from the backend instead, perhaps in a manner similar to #7675.